rasa/core/training/interactive.py

```diff
@@ -580,6 +580,21 @@ def _slot_history(tracker_dump: Dict[Text, Any]) -> List[Text]:
     return slot_strings
 
 
+async def _retry_on_error(
```

Comment:
why is this `async` @Dobatymo ?

Reply:
I am not really sure how `questionary` works. I saw that every function which uses it in this file is async so I made this async as well in case `questionary` requires some of the async functionality. If that is not the case, I will remove it.

Reply:
`async` isn't necessary in this case, I've added suggestions to remove it :+1: